__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+const connection = require('../db/connection');
+const request = require('supertest');
+const { forEach } = require('../db/data/test-data/articles');
+
```

Comment:
I'm unsure about what line 6 is requiring in ?

Reply:
i realised once I git push, is this automatic fill in vscode. you are right it is nothing.